packages/paypal-js/src/load-script.ts

```diff
@@ -19,6 +19,14 @@ export function loadScript(
     if (typeof document === "undefined") return PromisePonyfill.resolve(null);
 
     const { url, attributes } = processOptions(options);
+
+    if (
```

Comment:
I'd prefer to skip the validation on the existing value and only set it if the data attribute value is undefined. That way we can add new libraries in the future like vue-paypal-js and not have to worry about updating this code path.
```js
    if (!attributes["data-js-sdk-library"]) {
        attributes["data-js-sdk-library"] = "paypal-js";
    }
```

Reply:
Okay, cool. I'll need to look into this, because it ends up inserting a new script in the DOM even when there's already one with the same params.

Reply:
@gregjopa, I tried adding the `data-js-sdk-library` default value in `loadScript`, `processOptions`, `loadCustomScript`, and `insertScriptElement`, and ended up moving it back to `loadScript`. Definitely open to trying something else too, but marking this ready for review for now.

Reply:
I think you added it in the right spot in loadScript() @nbierdeman  💯